cmd/cronosd/cmd/fix-unlucky-tx.go

```diff
@@ -300,7 +301,7 @@ func newTxIndexer(config *tmcfg.Config, chainID string) (txindex.TxIndexer, erro
 		}
 		return es.TxIndexer(), nil
 	default:
```

Comment:
maybe better to use case "null" instead of default (which is supposed to be kv)

Reply:
https://github.com/tendermint/tendermint/blob/v0.34.x/node/node.go#L299
it match the tendermint's own logic though.